src/include/daos/dpar.h

```diff
@@ -9,10 +9,6 @@
 #include <stdbool.h>
 #include <inttypes.h>
 
-#ifdef __cplusplus
```

Comment:
why remove those?

Reply:
This is an internal header file so doesn't need c++ protection.

tse.h however does not have these guards in but I'm not sure if it's worth adding them.